core/src/main/java/jenkins/plugins/DetachedPluginsUtil.java

```diff
@@ -15,9 +17,12 @@
 import java.util.logging.Logger;
 import java.util.stream.Collectors;
 import java.util.stream.Stream;
+import jenkins.model.Jenkins;
 import org.apache.commons.io.IOUtils;
 import org.kohsuke.accmod.Restricted;
 import org.kohsuke.accmod.restrictions.NoExternalUse;
+import org.kohsuke.stapler.AncestorInPath;
+import org.kohsuke.stapler.QueryParameter;
 
 /**
  * Dedicated class to handle the logic related to so-called <em>detached plugins</em>.
```

Comment:
What does this new method have to do with detached plugins?

Reply:
It doesn't, I wasn't sure where to place the code at first actually